.circleci/config.yml

```
@@ -636,9 +636,15 @@ jobs:
           name: Run sample app tests
           command: |
             bash bin/run-ios-sample-app-test.sh
+            SIMULATOR_UDID=$(xcrun instruments -s devices | grep 'iPhone 11 (13' | awk -F '[][]' '{print $2}')
+            xcrun simctl spawn $SIMULATOR_UDID log collect --output ~/project/ios-integration-test.logarchive 2>/dev/null
```

TIL if there's only one simulator running, this works:

```suggestion
            xcrun simctl spawn booted log collect --output ~/project/ios-integration-test.logarchive 2>/dev/null
```

Then we don't even need the UUID

---

Worth a shot, I was just getting ready to try `fastlane scan`